demo/full-featured.md

```diff
@@ -11,6 +11,8 @@ keywords: example demo custom wysiwyg full-power full-featured plugins non-premi
 
 This example includes most of the available {{site.productname}} plugins, including plugins available on {{site.cloudname}} premium subscriptions. Please note, it includes [premium plugins]({{site.plugindirectory}}).
 
+Want to try it for yourself ? [Get Strated with TinyMCE now on Cloud]({{site.accountsignup}}).
```

Comment:
```suggestion
Want to try it for yourself ? [Get Started with TinyMCE now on Cloud]({{site.accountsignup}}).
```
also unsure about that capital? @tylerkelly13

Reply:
updated your suggestion to fix capitalisation.